samples/commands-and-operations/app/app.json

```diff
@@ -0,0 +1,44 @@
+{
+    "schemaVersion": "1.0",
+    "items": [
+        {
+            "$type": "menu",
+            "id": "iwtm-config",
+            "items": [
+                "item://menu-item/run-thing-1",
+                "item://menu-item/run-thing-2",
+                "item://menu-item/view-command-history"
+            ],
+            "title": "Execute Custom Behavior",
+            "isLandmark": true
+        },
+        {
+            "$type": "menu-item",
+            "id": "run-thing-1",
+            "title": "Run Command 1",
```

Comment:
When I first interacted with this example, I was initially confused when I clicked "Run Command 1" and "Run Command 2" and nothing at all happened. Even after I clicked "View Command History", it took me a while to connect what I saw in the dialog with the other two buttons. It wasn't until I went back and forth that I put it together. If I had seen the code and app config first, then it might have made more sense. But approaching the demo blind, it was a bit unintuitive. I wonder if there are some small tweaks we can make, like giving some sort of feedback when the first two options are selected (using ui.display-notification, for example)?

Reply:
I've updated this sample to something more compelling i think. I've also demonstrated the usage of `canExecute`.

Reply:
That's much better. Thanks!